### PR TITLE
sql-statements: use ENBF to render syntax diagrams for BACKUP and RESTORE

### PR DESCRIPTION
**Pull request by TomShawn**
…TORE

### What is changed, added or deleted? (Required)

Change the synopsis sections of BACKUP, RESTORE and SHOW {BACKUPS|RESTORES} to use the ` ```ebnf+diagrm` block introduced since pingcap/website-docs#83, replacing the original image-based sqlgrams (which files are deleted).

<details><summary>Output preview</summary>

1-fs8

</details>

The syntax diagrams are now rendered server-side as SVG when building (similar to SQLite3 / IBM DB2 / CockroachDB's docs), and user can also toggle between syntax diagram and the source EBNF (similar to Oracle's docs).

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4342
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-backup.md [hunk 1]
```diff
@@ -18,25 +18,24 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-backup/']
...
-**Boolean:**
-
-![Boolean](/media/sqlgram/Boolean.png)
-
-**BackupTSO:**
-
-![BackupTSO](/media/sqlgram/BackupTSO.png)
+```ebnf+diagram
+BackupStmt ::=
+    "BACKUP" BRIETables "TO" stringLit BackupOption*
```
Comment (ran-huang) [inline sql-statements/sql-statement-backup.md:23]: Will the blank lines affect the display?

**Review by ran-huang — COMMENTED** [review]
LGTM

File: sql-statements/sql-statement-backup.md (hunk 1 above)
Comment (kennytm) [reply sql-statements/sql-statement-backup.md:23]: No the treats new lines as the same as spaces which are ignored. Even

``````markdown
```ebnf+diagram
a ::= b c ::= d
```
``````

will work

**Review by kennytm — COMMENTED** [review]
LGTM but maybe you should fix the PR title :wink:

**Review by ran-huang — APPROVED** [review]

**Pull request merged**